packages/ui/src/components/editor/panels/Assets/container/index.tsx

```diff
@@ -222,7 +222,7 @@ const AssetPanel = () => {
 
   const CategoriesList = () => {
     return (
-      <div className="mb-8 h-[100%] w-[200px] overflow-y-auto bg-[#0E0F11] pb-8">
+      <div className="mb-8 h-[100%] w-[200px] overflow-y-auto bg-neutral-950 pb-8">
```

Comment:
can you please give me a link to this figma design?

Reply:
https://www.figma.com/design/Ds2NEvvz9nqddltP3lsKoP/Design-Changes?node-id=0-1&t=YLp6WBatFXncmfY7-0